types/js-clipper/index.d.ts

```diff
@@ -300,3 +300,6 @@ declare namespace ClipperLib {
 
 }
 
+declare ClipperLib = ClipperLib.Clipper;
```

Comment:
That won't work as is.
Just remove a namespace declaration (as all symbols are exported) and add:

```ts
export as namespace ClipperLib;
```
on the bottom. That would work like 'kind of' UMD module pattern, which this package seems to be (exports as script and as a module using custom factory block)

```diff
 //
-declare namespace ClipperLib {
[...]
-}

+export as namespace ClipperLib;
```

it can be used as CJS that way (via namespace):

```ts
import * as ClipperLib from 'js-clipper';
```

Reply:
it works your way. thank you so much. I still don't know the typescript world very well.